packages/forms/src/UIForm/fields/CheckBox/CheckBox.component.js

```diff
@@ -28,11 +28,11 @@ if (process.env.NODE_ENV !== 'production') {
 		id: PropTypes.string,
 		isValid: PropTypes.bool,
 		errorMessage: PropTypes.string,
-		onChange: PropTypes.func,
+		onChange: PropTypes.func.isRequired,
 		schema: PropTypes.shape({
 			description: PropTypes.string,
 			title: PropTypes.string,
-		}),
+		}).isRequired,
```

Comment:
schema is required and none of its component is, isn't it more safe then to rely on a default props set to an empty literral object ?